src/layers/core/hexagon-layer/hexagon-layer.js

```diff
@@ -238,7 +239,19 @@ export default class HexagonLayer extends CompositeLayer {
       getColor: this._onGetSublayerColor.bind(this),
       getElevation: this._onGetSublayerElevation.bind(this),
       updateTriggers: this.getUpdateTriggers()
-    });
+    };
+  }
+
+  getSubLayer() {
+    return HexagonCellLayer;
+  }
+
+  renderLayers() {
+    const SubLayerComp = this.getSubLayer();
```

Comment:
Can we change the name of this variable? `SubLayerComp` doesn't provide much info....